index.html

```diff
@@ -25,22 +25,28 @@
           <nav class="menu-bars">
             <span class="material-symbols-outlined"> menu </span>
           </nav>
+          <nav class="toolbar">
+            <a href="#">Portfolio</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </nav>
         </div>
       </header>
+
       <section class="main-content">
```

Comment:
Nice! but the bottom-left corner of the header layout is rounded, so please make sure to update it.

![image](https://user-images.githubusercontent.com/34779928/194421538-cdd6c968-f26e-40ba-a2c7-bcb0deef1f12.png)